src/evo/simplifiedmns.h

```diff
@@ -135,6 +144,10 @@ class CSimplifiedMNListDiff
     std::vector<std::pair<uint8_t, uint256>> deletedQuorums; // p<LLMQType, quorumHash>
     std::vector<llmq::CFinalCommitment> newQuorums;
 
+    // Map of Chainlock Signature used for shuffling per set of quorums
+    // The set of quorums is the set of indexes corresponding to entries in newQuorums
+    std::map<CBLSSignature, std::set<uint16_t>, BLSSignatureCompare> quorumsCLSigs;
```

Comment:
nit: why not lambda here?

Reply:
@knst I don't think a lamda would make the code cleaner necessary